MediaAdminBundle/Resources/public/template/Folder/folderElementTreeView._tpl.html

```diff
@@ -0,0 +1,34 @@
+<li data-folder-id="<%= folderTree.get('folder').get('id') %>"
+    <% if (true === folderTree.get('folder').get('rights').can_edit) {%> class="sortable-node"<% }%>
```

Comment:
It can't be changed as it used both by nodes and folder, via AbstractTreeView.js

That 'node' doesn't stand for OO node, but for tree node